avatar/avatar.css

```diff
@@ -0,0 +1,68 @@
+:root {
+  --primary-color: #bc3f4a;
+  --white: #f3f4f6;
```

Comment:
instead of --white, you could give a more descriptive name like --text-color-white

Reply:
Sure, will take care of that in the subsequent refactors.